src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -335,7 +335,7 @@ public bool IsProjectPackageReference
         {
             get
             {
-                return _nugetProjects.Any() && _nugetProjects.FirstOrDefault().ProjectStyle.Equals(ProjectModel.ProjectStyle.PackageReference);
+                return _nugetProjects.Any() && _nugetProjects.FirstOrDefault().ProjectStyle.Equals(ProjectModel.ProjectStyle.PackageReference) && !IsCentralPackageManagementEnabled;
```

Comment:
I believe if a project is PackageReference is independent of being CPM enabled. If you are using this property to show/hide the button, then, I would prefer another property to control whether or not the button is active.